### PR TITLE
[feat]: Adding Snackbar CSS

### PR DESCRIPTION
**Pull request by raghav2110**
Hi, I have added snackbar component. Please review my PR.

### CODE REVIEW

File: document-site.html [hunk 1]
```diff
@@ -69,6 +69,10 @@ <h2>Components</h2>
                     <li>
                         Text Utilities</li>
                 </a>
+                <a href="#snackbar" onclick="hideSideNavBar()">
```
Comment (junaidshaikh-js) [inline document-site.html:72]: Click event can be moved to JS file.

**Review by junaidshaikh-js — COMMENTED** [review]
Good work!

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: components/snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    background-color: var(--snackbarbg);
+    justify-content: space-between;
+    align-items: center;
+    margin: 2rem;
+    border-radius: 0.5rem;
+}
+
+.snackbar-left-comp{
+    padding: 1rem;
+    color: white;
```
Comment (AnuragSahu11) [inline components/snackbar/snackbar.css:34]: you can assign a variable to white color too, as a good practice

**Review by AnuragSahu11 — COMMENTED** [review]
Keep up the good work

File: components/snackbar/snackbar.css (hunk 2 above)
Comment (raghav2110) [reply components/snackbar/snackbar.css:34]: Sure will refactor that.

File: document-site.html (hunk 1 above)
Comment (raghav2110) [reply document-site.html:72]: Sure will refactor that.

**Pull request merged**